vector/src/main/res/xml/vector_settings_general.xml

```diff
@@ -12,6 +12,7 @@
             android:title="@string/settings_profile_picture" />
 
         <im.vector.app.core.preference.VectorEditTextPreference
+            android:enabled="false"
```

Comment:
Use a `VectorPreference` instead of `VectorEditTextPreference`, and keep enabled this field to display it correctly. Currently it is displayed in light grey instead of in primary color